docs/index.rst

```diff
@@ -2,19 +2,47 @@
 ops library API reference
 =========================
 
+The `ops` library is a Python framework for writing and testing Juju charms.
+
+  See more: `Charm SDK documentation <https://juju.is/docs/sdk>`_
+
+The library provides:
+
+- :ref:`ops_main_entry_point`, used to initialise and run your charm;
+- :ref:`ops_module`, the API to respond to Juju events and manage the application;
+- :ref:`ops_pebble_module`, the low-level API for Pebble in Kubernetes containers;
```

Comment:
The request from Dora was to list what the library contains, rather than what we want charmers specifically to focus on.

I've updated the language a bit.